client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -103,10 +104,7 @@ export const siteSetupFlow: Flow = {
 				case 'storeFeatures': {
 					const storeType = params[ 0 ];
 					if ( storeType === 'power' ) {
-						const args = new URLSearchParams();
-						args.append( 'back_to', `/start/setup-site/store-features?siteSlug=${ siteSlug }` );
-						args.append( 'siteSlug', siteSlug as string );
-						return redirect( `/start/woocommerce-install?${ args.toString() }` );
+						return navigate( 'storeAddress' );
```

Comment:
Could we possibly wrap this change in a feature flag? I'd like to be able to merge Woo PoC stuff without undermining the work you guys have done on making the existing Seller Experience steps ready for primetime.

(Alternatively and perhaps better - remove this chunk from this PR, and set up another PR that does just this chunk and adding the feature flag.)

Reply:
That makes sense. I'll do the separate PR with the feature flag.

Reply:
Updated so that branch is back to the original code from `trunk`.